LibiglOptions.cmake.sample

```diff
@@ -0,0 +1,72 @@
+# In order to persistently set default options for your project, copy this file
```

Comment:
This file is just a convenience to set default options when building libigl as a standalone. When including libigl as a subproject, just set options like before. But now rather than passing a slew of options via command-line when building libigl, you can use this non-versioned file as an alternative.

Reply:
I'm not sure I get the comparison to the current situation. Should I be looking at https://github.com/libigl/libigl-example-project/blob/next/CMakeLists.txt for an example of use? That looks pretty clean and doesn't seem to require copying this .sample file.

Reply:
There are two use-cases:
1. You are cloning https://github.com/libigl/libigl.git and want to compile/try out tutorials.
2. You are writing a project using libigl, following the model of https://github.com/libigl/libigl-example-project

For the first case, this sample option file offers a way to customize what modules are compiled, and whether we are compiling in static mode or header-only. This is mostly a convenience for libigl developers, but can be useful to end-users as well. Previously, the only way to set options for this use-case is to pass them via command-line, or set them via CMake-GUI/ccmake.

For the second case, there is no need to copy the sample file, as libigl options are controlled by the parent project ([this line](https://github.com/libigl/libigl-example-project/blob/23263bde8786f7454f90709f55aca460376ba9b6/CMakeLists.txt#L7) as you have pointed out).

Reply:
Ok, thanks, now I get it. Perhaps we can include this distinction in the comments/documentation if it's not already there.